x-pack/filebeat/module/mysqlenterprise/audit/test/mysql_audit_test.log-expected.json

```diff
@@ -0,0 +1,1100 @@
+        "event.type": [
+            "access"
+        ],
+        "fileset.name": "audit",
+        "input.type": "log",
+        "log.offset": 3019,
+        "mysqlenterprise.audit.class": "general",
+        "mysqlenterprise.audit.connection_id": 15,
+        "mysqlenterprise.audit.general_data.command": "Query",
+        "mysqlenterprise.audit.general_data.query": "GRANT ALL PRIVILEGES ON *.* TO 'root'@'hades.home' IDENTIFIED BY 'password'",
+        "mysqlenterprise.audit.general_data.sql_command": "grant",
+        "mysqlenterprise.audit.general_data.status": 1064,
+        "mysqlenterprise.audit.id": 0,
+        "mysqlenterprise.audit.login.user": "root",
+        "related.user": [
+            "root"
+        ],
+        "server.user.name": "root",
+        "service.type": "mysqlenterprise",
+        "tags": [
+            "mysqlenterprise-audit"
+        ]
+    },
+    {
+        "@timestamp": "2020-10-01T19:28:54.000Z",
+        "client.domain": "localhost",
+        "event.action": "mysql-status",
+        "event.category": [
+            "database"
+        ],
+        "event.dataset": "mysqlenterprise.audit",
+        "event.kind": "event",
+        "event.module": "mysqlenterprise",
+        "event.outcome": "failure",
+        "event.type": [
+            "access"
+        ],
+        "fileset.name": "audit",
+        "input.type": "log",
+        "log.offset": 3410,
+        "mysqlenterprise.audit.class": "general",
+        "mysqlenterprise.audit.connection_id": 15,
+        "mysqlenterprise.audit.general_data.command": "Query",
+        "mysqlenterprise.audit.general_data.query": "GRANT ALL PRIVILEGES ON *.* TO 'root'@'%'",
+        "mysqlenterprise.audit.general_data.sql_command": "grant",
+        "mysqlenterprise.audit.general_data.status": 1410,
+        "mysqlenterprise.audit.id": 0,
+        "mysqlenterprise.audit.login.user": "root",
+        "related.user": [
+            "root"
+        ],
+        "server.user.name": "root",
+        "service.type": "mysqlenterprise",
+        "tags": [
+            "mysqlenterprise-audit"
+        ]
+    },
+    {
+        "@timestamp": "2020-10-01T19:29:36.000Z",
+        "client.domain": "localhost",
+        "event.action": "mysql-status",
+        "event.category": [
+            "database"
+        ],
+        "event.dataset": "mysqlenterprise.audit",
+        "event.kind": "event",
+        "event.module": "mysqlenterprise",
+        "event.outcome": "failure",
+        "event.type": [
+            "access"
+        ],
+        "fileset.name": "audit",
+        "input.type": "log",
+        "log.offset": 3767,
+        "mysqlenterprise.audit.class": "general",
+        "mysqlenterprise.audit.connection_id": 15,
+        "mysqlenterprise.audit.general_data.command": "Query",
+        "mysqlenterprise.audit.general_data.query": "CREATE USER 'audit_test_user'@'localhost' IDENTIFIED BY <secret>",
+        "mysqlenterprise.audit.general_data.sql_command": "create_user",
```

Comment:
should this be categorized with `iam` values?

Reply:
I think the issue with "sql_command" is that we would have to create a list of conditions and trying to catch all the different SQL commands it supports. 
I can try to add a few at least!